setup.py

```diff
@@ -11,76 +11,68 @@
 # License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 # either express or implied.  See the License for the specific language governing permissions
 # and limitations under the License.
-#--------------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------------------------------
 
 import os
-from distutils.core import *
-from subprocess import *
+import platform
+import subprocess
 
-package_name = 'Tkrzw'
+from setuptools import Extension, setup
+
+package_name = 'tkrzw'
```

Comment:
clarification: Is it mandatory for the package name to be lower case?
just curiocity: Do you now now we should treat multiple words in the pip name, "foo_bar", "foo-bar", "foobar", or "fooBar"?

Reply:
We can keep it titled (I know Flask uses this syntax). I’ll update.
I think foo-bar is most used but I’m not aware of strict rules.